flocker/node/agents/shadow_bind_mounter.py

```diff
@@ -0,0 +1,31 @@
+# -*- test-case-name: flocker.node.agents.test.test_shadow_bind_mounter -*-
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+from .blockdevice_manager import Permissions
+
+
+def create_tmpfs_shadow_mount(backing_directory, read_only_directory,
+                              blockdevice_manager):
+    """
+    Creates a "shadow mount directory". This is a tmpfs mount that is bind
+    mounted read-only to another location. The intent is to have the following
+    characteristics:
+
+        - The read_only_directory is cleared on reboots.
+        - The read_only_directory can be changed only by agents that know the
+            path of the backing directory.
```

Comment:
It's worth adding a 'so that' for each of these characteristics, explaining why each is desirable. (I can guess the second, but not the first)